ext/data/schemas/dictionary-index-schema.json

```diff
@@ -47,6 +47,14 @@
             "type": "string",
             "description": "Attribution information for the dictionary data."
         },
+        "sourceLanguage": {
+            "type": "string",
+            "description": "Language of the terms in the dictionary."
```

Comment:
iso format or does that not matter?

Reply:
I considered a pattern like `^[a-z]{2,3}-[A-Z]{2}$`, but I haven't been able to find a perfect one.
Or we could do a simple pattern with all our currently supported languages like `en|de|fa|fi...`, but that's another thing to tweak every time a new language is added. 
Or use this https://gist.github.com/konsorten-michael/094f898c3a2847c642a920f7c6f4f575

Reply:
Let's use the gist? People can submit issues if something else comes up. If we want to support all languages handlebars i feel it's important to enforce all the same languages use the same string for source language

Reply:
I'll throw the gist in, just wanna note that it contains ISO 639-1 codes, we might need to add to that over time for languages that are only in 639-2 or 3, e.g. we have grc as an option in the dropdown.